pkg/bucketaws/setup.go

```diff
@@ -113,12 +113,15 @@ func DownloadFromBucket(svc *s3.S3, dwn *s3manager.Downloader, dates []string, b
 				if strings.Contains(*item.Key, x) {
 					fooFile, fileName := files.FilePathSetup(item.Key, dwn)
 					logrus.Info("Downloading ", fileName)
+					start := time.Now()
 					_, err := dwn.Download(
 						fooFile,
 						&s3.GetObjectInput{
 							Bucket: aws.String(viper.GetString("bucket")),
 							Key:    aws.String(*item.Key),
 						})
+					duration := time.Since(start)
+					logrus.Info("Download took: ", duration.Round(10000000))
```

Comment:
I would advise against having naked numbers like this. Personally, at first glance I find it difficult to tell to which precision will it be rounded to. Maybe use `time.Second`, `time.Minute`, or similar?

Reply:
Good point, changed.